src/components/Menu/menu.scss

```diff
@@ -88,3 +88,10 @@
 .sb-menu-button-borderless {
   border: 0;
 }
+
+.sb-menu-item--is-link {
+  width: initial;
+  padding: 9px 20px;
```

Comment:
It has to be `9.25px 20px` otherwise the links will keep having 32px of height and buttons 32.5px

Reply:
Thanks for the help @fariacc , you is the best 💪🏼